src/lib/detection_algorithms/main.js

```diff
@@ -0,0 +1,50 @@
+//read from config to find detection algorthims and paths
```

Comment:
Got it I thought this was more organized but I will respect the conventions what you are saying makes sense